src/SI_Toolkit/Predictors/predictor_ODE_tf.py

```diff
@@ -43,13 +43,8 @@ def __init__(self, horizon=None, dt=0.02, intermediate_steps=10, disable_individ
         self.dt = dt
         self.intermediate_steps = intermediate_steps
 
-        if planning_environment is None:
-            self.next_step_predictor = next_state_predictor_ODE_tf(dt, intermediate_steps, self.batch_size,
-                                                                   disable_individual_compilation=True)
-        else:
-            self.next_step_predictor = next_state_predictor_ODE_tf(dt, intermediate_steps, self.batch_size,
-                                                                   disable_individual_compilation=True,
-                                                                   planning_environment=planning_environment)
```

Comment:
We stopped using this concept of planning_environment. The `else` condition was never hit anymore